### PR TITLE
Improve build instructions

### LINKED ISSUE #166: Improve build instructions
docs/build.md:

* Should mention pull-upstream.sh.
* Should list debhelper as a prerequisite (used to build source packages).
* Should list sudo as a prerequisite (used by build-acis).
* Should list python3 as a prerequisite (used by build-acis in a oneliner to list nameservers, though that should maybe just use awk or something).
* Should list systemd-container as a prerequisite (acbuild uses systemd-nspawn by default).
* Should maybe list debootstrap as a prerequisite (used by build-acis, though sbuild recommends it).
* Should not list cpio, genisoimage as prerequisites (they’re pulled in as needed inside the sbuild chroot, but not used outside).
* `homeworld-rkt-<newest>.deb` should be `homeworld-rkt_<newest>.deb`.
* After `dpkg -i homeworld-rkt_<newest>.deb`, run `apt install -f` to resolve any dependency problems.
* `//building` is a confusing metasyntax (or typo?), maybe use relative paths instead?

### PR DESCRIPTION
**Pull request by andersk**
Includes #154. Fixes #166.

### CODE REVIEW

File: building/build-acis/common/container-build-helpers.sh [hunk 1]
```diff
@@ -224,7 +224,7 @@ function run_builder() {
 	RKT_OPTS+=(--volume "build,kind=host,source=${BUILDDIR},readOnly=false")
 	RKT_OPTS+=(--mount volume=build,target=/build)
 
-	DNS_ADDR="$(python3 -c 'with open("/etc/resolv.conf") as f: print(*[line.split(" ")[1].strip() for line in f if line.startswith("nameserver ")][:1])')"
+	DNS_ADDR="$(awk '$1=="nameserver" {print $2; exit}' /etc/resolv.conf)"
```
Comment (celskeggs) [inline building/build-acis/common/container-build-helpers.sh]: Very nice.

File: docs/build.md [hunk 2]
```diff
@@ -3,48 +3,58 @@
 Install packages:
 
  * build-essential
+ * debhelper
+ * debootstrap
  * sbuild
- * cpio
- * genisoimage
+ * sudo
+ * systemd-container
```
Comment (celskeggs) [inline docs/build.md]: cpio and genisoimage are required if someone wants to directly build spire, which isn't uncommon.

Comment (andersk) [reply docs/build.md]: Building spire requires `zip`, `python3-yaml`, and `libarchive-tools`, but `cpio` and `genisoimage` are only needed at runtime. How do you want this to be handled? Are all of spire’s build-time and runtime dependencies to be considered build prerequisites, or only its build-time dependencies, or only what’s needed to make the standard sbuild process work?

Comment (celskeggs) [reply docs/build.md]: Hmm, we probably just care about {build time dependencies} plus {sbuild dependencies}.

Comment (andersk) [reply docs/build.md]: Cared.

**Pull request merged**